docs/source/users/memory-game.rst

```diff
@@ -0,0 +1,61 @@
+Memory game
+-----------
+
+For the purpose of not disclosing to participants that language attitudes are being measured,
+a memory game was implemented to act as a distractor after the completion of linguistic tasks
```

Comment:
Change this to present tense. E.g. ... a task implementing a simple memory card game *is provided* to act as a distractor